docs/make.jl

```diff
@@ -12,7 +12,7 @@ makedocs(sitename = "MethodOfLines.jl",
          authors = "Chris Rackauckas, Alex Jones et al.",
          clean = true, doctest = false, linkcheck = true,
          modules = [MethodOfLines],
-         warnonly = [:docs_block, :missing_docs, :cross_references],
+         warnonly = true,#[:docs_block, :missing_docs, :cross_references],
```

Comment:
You can instead turn on debug mode. https://github.com/SciML/DiffEqFlux.jl/blob/master/.buildkite/pipeline.yml#L39C20-L40C37

Reply:
This change was reverted, and the suggestion of setting `JULIA_DEBUG="Documenter"` was taken.